lib/components/viewers/viewers.css

```diff
@@ -103,6 +103,112 @@ a.routeDetails {
   outline: 0px;
 }
 
+/* child stop details */
+
+.otp .stop-viewer .stop-viewer-body .stop-label  {
+  display: table;
+  width: 100%;
+}
+
+.otp .stop-viewer .stop-viewer-body .stop-label .stop-name  {
+  display: table-cell;
+  max-width: 300px;
+}
+
+.otp .stop-viewer .stop-viewer-body .stop-label .view-child-stop-button  {
+  color: #000080;
+  border: none;
+  background: none;
+  display: table-cell;
+  padding: 0px;
+  width: 90px;
+}
+
+.otp .stop-viewer .stop-viewer-body .related-panel-title  {
+  background-color: #ffffff;
+  padding: 0.5rem;
+  position: relative;
+  width: fit-content;
+  margin-left: 1rem;
+}
+
+.otp .stop-viewer .stop-viewer-body .related-panel-container  {
+  border: 1px solid;
+  border-radius: 7px;
+  padding: 0.5rem;
+  align-items: center;
+  margin-top: -2.5rem;
+}
+
+.otp .stop-viewer .stop-viewer-body .related-items-list  {
+  padding-top: 20px;
+  width: 100%;
+  margin: 0;
+}
+
+.otp .stop-viewer .stop-viewer-body .related-item  {
+  padding: 1rem;
+  margin-bottom: 1rem;
+}
+
+.otp .stop-viewer .stop-viewer-body .related-item .item-label  {
+  display: flex;
+  width: 100%;
+  justify-content: flex-start;
```

Comment:
Is it possible to align items so the text and icon are aligned? `align-items: baseline` should do this